src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -10,7 +10,7 @@ namespace Internal.Runtime.CompilerHelpers
     /// <summary>
     /// These methods are used to throw exceptions from generated code.
     /// </summary>
-    internal static class InteropHelpers
+    public static class InteropHelpers
```

Comment:
Okay I can make that work; all methods being compiled regardless of type visibility. The wrinkle is that S.P.CoreLib has code coverage goo (CoverageSupport.cs) with an eager constructor that P/Invokes into some Magellan DLL. I can provide a stub in our bootstrap code that returns null for it to satisfy the dangling dependency. Lemme see what I can work up for that.